libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1365,6 +1384,34 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 		});
 	}
 
+	if (_from.category() == Type::Category::ArraySlice)
+	{
```

Comment:
Also better to assert that ``_from.isDynamicallySized()`` here.

Reply:
Fixed